platform/bootstrap/plugins-swarm.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
a way to keep a single plugins.json file would be to declare two plugins for manager: manager-os and manager-swarm, these are just identifiers, the Cmd value effectly names the plugin.